### ISSUE TITLE
Typo in residue.js

### ISSUE THREAD

**Issue by paulsmirnov** [biasmv/pv#129]
Hi! It seems that there's a mistake in a condition:

```
index 70b137c..f1dcfff 100644
--- a/src/mol/residue.js
+++ b/src/mol/residue.js
@@ -69,7 +69,7 @@ ResidueBase.prototype = {
       }
       return null;
     }
-    if (index_or_name >= this._atoms.length && index_or_name < 0) {
+    if (index_or_name >= this._atoms.length || index_or_name < 0) {
       return null;
     }
     return this._atoms[index_or_name];
```

**Comment by biasmv** [comment]
Thanks, yes you are right. I'll fix it when I'm back home.

**Comment by biasmv** [comment]
Thanks again for the report. I've fixed it in cd54434 and added a small test in d99d2bd.
